packaging_automation/tests/files/packaging_warning/sample_warning_build_output_deb.txt

```diff
@@ -129,13 +129,15 @@ gpg: Total number processed: 1
 gpg:               imported: 1  (RSA: 1)
 gpg:       secret keys read: 1
 gpg:   secret keys imported: 1
-Warnings generated by release docker build :
```

Comment:
I would not want to lose this line here

Reply:
I need to because this is not the starter line for lintian checks any more you can check it below
https://github.com/citusdata/packaging/runs/3619678886?check_suite_focus=true#step:6:3035